### PR TITLE
[8.0][FIX] fix account_prepayment

### PR DESCRIPTION
**Pull request by noahzaozao**

### CODE REVIEW

**Comment by noahzaozao** [comment]
@Elico-Corp/odoo-reviewers

File: account_prepayment/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class res_partner(osv.osv):
+    _inherit = 'res.partner'
+
+    _columns = {
+        'property_account_prepayable': fields.many2one(
+            'account.account',
+            string="Account Payable (Prepayment)",
+            domain="[('type', '=', 'payable')]",
+            help="This account will be used instead of the default one as the prepayment payable account for the current partner",
```
Comment (elicoidal) [inline account_prepayment/models/res_partner.py]: Can you make it PEP8 compliant?

**Review by elicoidal — APPROVED** [review]
:+1:

**Comment by noahzaozao** [comment]
@Elico-Corp/odoo-reviewers Done , please review this PR

**Comment by seb-elico** [comment]
LGTM
@Elico-Corp/odoo-reviewers please review and merge if ok

**Pull request merged**